services/cluster-agent/tests/test_run.py

```
@@ -0,0 +1,138 @@
+
+from unittest.mock import AsyncMock, patch
+
+import httpx
+import pytest
+
+from cluster_agent import ingest_client, run
+from cluster_agent.backoff import Backoff
+
+
+@pytest.fixture(autouse=True)
+def _reset_shutdown_event():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_reset_shutdown_event`

(ANN202)